src/phaser-game/scenes/FortNerf.ts

```diff
@@ -1,168 +1,198 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/no-shadow */
 import Phaser from 'phaser';
+import { BULLET_MOVEMENT, BULLET_OFFSET } from '../utils/constants';
 
-// let platforms;
-let map: any;
+let background: any;
```

Comment:
Looooottttttssssss of `any`s. Definitely type these properly as soon as you can.